internal/packages.bzl

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
my style convention has been to name the file after the rule it exports, so this should be
npm_install.bzl